news/_posts/2019-05-29-severity-classification.md

```diff
@@ -0,0 +1,103 @@
+- A display problem that prevents you to do something (you can’t click on an element)
+- An alert message displayed in your back-office that it is possible to close
+- The product doesn't get added to cart after deleting product customisation
+- When cloning a product, carriers information is not copied
+- Statistics graph not working or showing wrong data 
+
+
+### Trivial
+
+The bug doesn’t affect any functionality or data. It does not impact productivity or efficiency. It is only an inconvenience without functional impact and it does not even need a workaround.
+
+Examples:
+
+- Cosmetic issues
+- Wrong translation in a specific language: that can be solved on [Crowdin](https://crowdin.com/project/prestashop-official)
+- Missing confirmation message after an action
+- A link opened in the same tab instead of a new tab
+
+## Issue Prioritization
+
+Assessing severity helps to prioritize issues but it is not the only criterion at stake. Given two equally severe issues, how to choose one over the other ?
+
+At PrestaShop, prioritization is done by representatives of the Development team, the Product Management team, and the Quality Assurance team. Together, during regular meetings, we look at the new confirmed issues and we sort them.
+
+One of the things we value a lot is making sure that a given bug does not besmirch PrestaShop’s image nor it affects the confidence merchants can have in PrestaShop. This factor is really important as it can durably damage the adoption of the project and its long-term survival. 
+
+We take special care and strive to make every version of PrestaShop better than the one before. Since no one wants to introduce new bugs while fixing other bugs, regressions are usually prioritized higher than older bugs. By doing this, we can assure that the overall software stability is ever increasing. 
+
+In addition, we also study the technical complexity that is, whether the bug is easy or more complex to solve. In some cases we will choose to prioritize a smaller bug over a bigger one, not because we are lazy, but because a complex bug may require big technical changes which are not suitable until a later version. Because it requires backwards-incompatible changes (which are bad for module developers), or because it can be better addressed as a part of a bigger project – for example. There is no use fixing a bug if the whole feature is due to be revamped in the near future. Speaking of which, sometimes we will prioritize some bugs just because of the “opportunity cost” of fixing them together, as it’s usually easier to fix several bugs within the same component. For example, during the migration of a page to Symfony, the bugs of this page are prioritized in order to fix them.
+
+Finally, we pay attention to the business impact. Since we care a lot about PrestaShop merchants, we are eager to find a solution when it affects their sales proceeds. Or ours. After all, we need to eat as well, and we use PrestaShop too.
```

Comment:
```suggestion
Finally, we pay attention to the business impact. Since we care a lot about PrestaShop users (merchants, agencies, developers ...), we are eager to find a solution when it affects their sales proceeds. Or ours. After all, we need to eat as well, and we use PrestaShop too.
```